_docs/cardano/2017-01-16-updating.md

```diff
@@ -63,7 +63,7 @@ The fact that stakeholders are responsible for system updates does not restrict
 us to a system where every single update requires a signature from the majority
 of stake. We can introduce the concept of an **implicit agreement**.
 
-An update has to have at least `T%` of the stake signatures to be published on
+An update has to have at least minimal bound (`updateProposalThd` in configuration.yaml) of the stake signatures to be published on
```

Comment:
> `updateProposalThd` in configuration.yaml

Please add a link to this configuration file: 

```
[configuration.yaml](https://github.com/input-output-hk/cardano-sl/blob/e26ad11397c87ce8b00f2a26d5e237f54d6ea90a/lib/configuration.yaml).
```

Reply:
I'm sure it would be more convenient for the reader to know where this configuration file lives. :-)